gtda/time_series/tests/test_target.py

```diff
@@ -51,4 +51,4 @@ def test_labeller_invalid_percentage():
     labeller = Labeller(width=5, func=np.max, func_params={},
                         percentiles=[101], n_steps_future=2)
     with pytest.raises(ValueError):
-        labeller.fit_transform_resample([], [])
+        labeller.fit_transform_resample(X, signal)
```

Comment:
Weren't we checking that `ValueError` is raised with empty lists here? Is this no longer what we want?

Reply:
As the name  of the test suggests, I wanted to test whether the percentage is in the valid range. Ofc, it was throwing in a `ValueError`  anyway, but for different reasons. Perhaps two tests, one for empty list, the other for invalid percentage?

Reply:
I see, so the test was previously misnamed perhaps :-) happy with the current state.